ios/FluentUI/Navigation/BadgeLabelButton.swift

```diff
@@ -53,6 +57,9 @@ class BadgeLabelButton: UIButton {
         static let badgeBorderWidth: CGFloat = 2
         static let badgeHorizontalPadding: CGFloat = 10
         static let badgeCornerRadii: CGFloat = 10
+
+        static let maximumContentSizeCategory: UIContentSizeCategory = .extraExtraLarge
+        static let minimumContentSizeCategory: UIContentSizeCategory = .large
```

Comment:
Where do these values come from?

Reply:
These were found through observation to be consistent with what UIKit does by default. I'll add a comment for this.

Reply:
Do we want to be behaving the same as UIKit? What does design say about what these defaults should be?

Reply:
Design hasn't given us any specs for text sizes in any content size categories besides [`.large`](https://developer.apple.com/documentation/uikit/uicontentsizecategory/1622934-large), so keeping consistency with UIKit is the best we can do.